content/Solutions/Integrations/Veeam.md

```diff
@@ -107,7 +107,7 @@ Recommended sizing:
   The number of VMs is based upon conservative throughput estimates with an average VM size set as 100GB and a backup window of eight hours running full backups.
   All other requirements for the number of Veeam backup proxies, and networking dependencies also apply.
 
-* Number of VMs Backed Up. 
+* Number of VMs Backed Up.
```

Comment:
if using as a bullet point we don't need to capitalize each word like a heading.
```suggestion
* Number of VMs backed up.
```